spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+
+  before(:each) do
+      @photo = create(:photo)
+      @comment = @photo.comments.create(body: "New Comment")
```

Comment:
Use of @comment in this before each method is redundant. I only need a photo, a user and a session with the user id